Source/Core/CesiumTerrainProvider.js

```diff
@@ -417,11 +425,19 @@ define([
 
         var skirtHeight = provider.getLevelMaximumGeometricError(level) * 5.0;
 
+        var rectangle = provider._tilingScheme.tileXYToRectangle(x, y, level);
+        var boundingOBB;
+        if (rectangle.width < CesiumMath.PI_OVER_TWO + 0.001) {
```

Comment:
Use the epsilon values in `Math`.  You might want to also add a comment as to why we're using the epsilon as it wasn't clear to me at first glance (so we don't skip a level it looks like).

Also, shouldn't we check the `height` too?  It is probably a bad long-term move to assume `height <= width`.